addons/account/models/ir_module.py

```diff
@@ -62,7 +62,7 @@ def write(self, vals):
         was_installed = len(self) == 1 and self.state in ('installed', 'to upgrade', 'to remove')
         res = super().write(vals)
         is_installed = len(self) == 1 and self.state == 'installed'
-        if not was_installed and is_installed and not self.env.company.chart_template and self.account_templates:
+        if not was_installed and is_installed and not self.env.company.chart_template and self.account_templates and (self.name != 'account' or config['test_enable']):
```

Comment:
Well don't want to effect all the current test case

Though `partner_autocomplete` test is failing (already known error) but not so sure if this commit is effecting it,
because yesterday one build passed all test cases, before making changes here :thinking: